flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
why limiting architectures?

Reply:
Because ARM isn't supported.

Reply:
but it's written in Java, isn't it supposed to run everywhere?

Reply:
It uses several native libraries. I haven't gone through the process of testing the use of these on ARM.

Reply:
I'm also not sure if Compose Multiplatform (the UI framework) even supports ARM on Linux.